commitlint.config.ts

```diff
@@ -167,6 +173,34 @@ module.exports = {
                     ]
                 },
 
+                'footer-correct-place': ({body}: {body:any}) => {
+                    let offence = false;
+
+                    if (body !== null) {
+                        let bodyStr = convertAnyToString(body, "body");
+
+                        let seenBody = false;
+                        let seenFooter = false;
+                        let lines = bodyStr.split(/\r?\n/);
+                        for (let line of lines) {
+                            if (isEmptyLine(line)){
+                                continue;
+                            }
+                            seenBody = seenBody || !isFooterNote(line);
+                            seenFooter = seenFooter || isFooterNote(line);
+                            if (seenFooter && !isFooterNote(line)) {
+                                offence = true;
+                                break;
+                            }
+                            
+                        }
+                    }
+                    return [
+                        !offence,
+                        `Please add footer messages after body paragraphs`
```

Comment:
@parhamsaremi this error message starting with "Please add" would make a developer think that he has to "add stuff" (as in, make the commit message bigger), but this is not what this rule should advice, the thing to be done is to shuffle the content around, so let's not use the word "add" here, it's confusing